x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/operator/MultivalueDedupeBytesRef.java

```diff
@@ -317,6 +340,25 @@ private void writeSortedWork(BytesRefBlock.Builder builder) {
         builder.endPositionEntry();
     }
 
+    /**
+     * Writes a {@link #work} to a {@link BytesRefBlock.Builder}.
+     */
+    private void writeWork(BytesRefBlock.Builder builder, boolean ascending) {
+        if (w == 1) {
+            builder.appendBytesRef(work[0]);
+            return;
```

Comment:
I don't think you need this one. You don't end up in this case with `w == 1`. That may be true for the method I wrote above too.

Reply:
I'll remove them from the deduplicatedSortedWork too.